### PR TITLE
Feat/store gender info

### PR DESCRIPTION
**Pull request by riz671**
1. Refactor Home Screen to React Hooks.
2. Added gender fetching logic for redux and on update.

### CODE REVIEW

File: navigation/MainNavigator.js [hunk 1]
```diff
@@ -30,9 +29,9 @@ function MainNavigator() {
             case "Profile":
               iconName = "ios-contact";
               break;
-            case "Playground":
-              iconName = "md-happy";
-              break;
+            // case "Playground":
```
Comment (kokosaka) [inline navigation/MainNavigator.js:32]: Still want to keep it? or delete all playground?

File: redux/actions/authActions.js [hunk 2]
```diff
@@ -1,34 +1,64 @@
...
+    payload: gender,
+  };
+};
+
+const fetchGender = (name) => {
   return (dispatch) => {
-    dispatch(fetchUser());
+    firebase
+      .database()
+      .ref(`users/${name}`)
```
Comment (kokosaka) [inline redux/actions/authActions.js:40]: 👍

**Review by kokosaka — APPROVED** [review]
looks great.

File: navigation/MainNavigator.js (hunk 1 above)
Comment (riz671) [reply navigation/MainNavigator.js:32]: It served it's purpose. I'm okay with deleting all code related to playground in another PR.

**Pull request merged**